docs/user/algo_bc.md

```diff
@@ -0,0 +1,66 @@
+# Behavioral Cloning
+
+```eval_rst
+.. list-table::
+   :header-rows: 0
+   :stub-columns: 1
+   :widths: auto
+
+   * - **Paper**
+     - Model-Free Imitation Learning with Policy Optimization :cite:`ho2016model`
+   * - **Framework(s)**
+     - .. figure:: ./images/pytorch.png
+        :scale: 10%
+
+
+        PyTorch
+   * - **API Reference**
+     - `garage.torch.algos.BC <../_autoapi/garage/torch/algos/index.html#garage.torch.algos.BC>`_
+   * - **Code**
+     - `garage/torch/algos/bc.py <../_modules/garage/torch/algos/bc.html#BC>`_
```

Comment:
looks like this link is broken on ReadTheDocs -- just use the direct github link for now.